gems/pending/openstack/events/openstack_rabbit_event_monitor.rb

```diff
@@ -37,12 +37,17 @@ def self.test_connection(options = {})
       connection = connect(options)
       connection.start
       return true
+    rescue Bunny::AuthenticationFailureError => e
+      $log.info("MIQ(#{name}.#{__method__}) Failed testing rabbit amqp connection: #{e.message}")
```

Comment:
Maybe do logging cleanup later.  Looks like we have lots of logging around these types of failures.  And, we need to clean up the logs around these validations.

We can do that cleanup in a separate PR.